src/bci_build/package/__init__.py

```diff
@@ -1498,15 +1501,15 @@ def build_tags(self) -> list[str]:
         for name in [self.name] + self.additional_names:
             tags += [
                 f"{self._registry_prefix}/bci-{name}:%OS_VERSION_ID_SP%",
-                f"{self._registry_prefix}/bci-{name}:{self.version_label}",
+                f"{self._registry_prefix}/bci-{name}:{self.image_ref_name}",
```

Comment:
Any particular reason why we shouldn't or don't want to use `oci_version` here?

Reply:
it's unlikely that `oci_version` gets reimplemented in a derived class. however `image_ref_name` might, because it is "fixed" tag that this image should be identified with. hence I think it makes sense to use that one as the primary tag, which has an important meaning for the build service (and the buildname/buildversion logic)